contrib/perftest/main.go

```diff
@@ -0,0 +1,261 @@
+package main
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"os"
+	"time"
+
+	"github.com/containers/image/types"
+	"github.com/containers/libpod/libpod"
+	image2 "github.com/containers/libpod/libpod/image"
+	cc "github.com/containers/libpod/pkg/spec"
+	"github.com/containers/libpod/pkg/util"
+	"github.com/containers/storage/pkg/reexec"
+	"github.com/cri-o/ocicni/pkg/ocicni"
+	"github.com/pkg/profile"
+	"github.com/sirupsen/logrus"
+)
+
+/*
+const (
+	testImageName = "docker.io/library/alpine:latest"
+	testRunCount  = 50
+)*/
+
+var helpMessage = `
+-count int
+	count of loop counter for test (default 50)
+-image string
+	image-name to be used for test (default "docker.io/library/alpine:latest")
+`
+
+func main() {
+
+	ctx := context.Background()
+	imageName := ""
+
+	testImageName := flag.String("image", "docker.io/library/alpine:latest", "image-name to be used for test")
+	testRunCount := flag.Int("count", 50, "count of loop counter for test")
```

Comment:
I see the help message for the count, but it looks like you've 50 hard coded here?  Is that coming with future changes?

Reply:
50 is default value, will be overridden if value is provided for flag.
e.g ``# perftest --count 10``

Reply:
Maybe make it a constant like defaultCount, so that we don't have magic numbers floating around.

Reply:
@kunalkushwaha right, but I was expecting to see a variable like inputCount here in line 50 rather than 50.